content/doc/book/blueocean/pipeline-editor.adoc

```diff
@@ -18,130 +18,115 @@ endif::[]
+Here, you can modify the name of the stage, delete the stage, and add steps to a stage.
 
-image:blueocean/editor/stage-configuration.png[Stage Configuration, role=center]
+image:blueocean/editor/stage-configuration.png[Stage configuration pane, role=center]
 
-The name of the Stage can be set at the top of the Stage Configuration sheet.
-The context menu (three dots on the upper right), can be used to delete the current stage.
-Clicking "Add step" will display the list of available Steps types with a search bar at the top.
-Steps can be deleted using the context <<step-configuration, context menu in the step configuration sheet>>.
-Adding a step or selecting an existing step will open the <<step-configuration, step configuration sheet>>.
+The name of the stage can be set at the top of the stage configuration pane.
+The more menu, represented by three dots to the right of the stage name, allows users to delete the currently selected stage.
+Selecting *Add step* displays the list of available step types.
+After selecting a step type, the page displays the step configuration pane.
 
 image:blueocean/editor/step-list.png[Step list filtered by 'file', role=center]
 
 == Step configuration
 
-Selecting a step from the Stage configuration sheet will open the Step Configuration sheet.
+This pane displays differently depending on the step type selected, and contains the fields or controls necessary for it.
 
 image:blueocean/editor/step-configuration.png[Step configuration for JUnit step, role=center]
 
-This sheet will differ depending on the step type,
-containing whatever fields or controls are needed.
-The name of the Step cannot be changed.
-The context menu (three dots on the upper right), can be used to delete the current step.
-Fields that contain incomplete or invalid information will display a warning symbol.
-Pipelines can have validation errors while they are being edited,
-but cannot be saved until the errors are fixed.
+After saving the name of the step, it will remain as originally entered.
+The more menu, represented by three dots to the right of the step name, allows users to delete the current step.
+Fields that contain incomplete or invalid information display a icon:exclamation[].
+Pipelines can have validation errors while editing, but saving is unavailable until there are no errors.
 
 image:blueocean/editor/step-error.png[Step configuration with error, role=center]
 
 
 == Save Pipeline dialog
 
-In order to be run, changes to a Pipeline must be saved in source control.
-The "Save Pipeline" dialog controls the saving of changes to source control.
+Changes to a Pipeline must be saved in source control before running.
+The *Save Pipeline* dialog controls the saving of changes to source control.
+
+image:blueocean/editor/save-pipeline.png[Save Pipeline dialog, role=center]
+
+There is a field to enter a description of the changes, but leaving this blank does not prevent the run.
```

Comment:
I was scolded very sternly from my writing instructor years ago to never start a lead in with "there is" or "there are". She had no reference for it, she said "just don't do it". I understand that now, since there are other ways of saying that there is something on the page, but I never know how to provide feedback for this when it happens.  Sorry it is a weird rationale! 😄 
```suggestion
You can optionally enter a description of the changes before saving.
```

Reply:
I'm not as strict as Cari's writing instructor (lol), but agree that these words are often unnecessary (at best) and at worst they can end up as garden-path sentences. Besides those words, Cari's rewrite is strong b/c it addresses the user in 2nd person and is active voice.

Reply:
I love it, that makes way more sense and definitely aligns with not wanting to leave out any information or make it disconnected in any way. Thanks!!